docs/src/pages/Conformance.md

```diff
@@ -69,17 +69,30 @@ By default, the returned bundle will contain up to 100 resource references. The
 
-To guarantee no data is skipped, clients should not process resources with a last_updated timestamp which is after `{current-time} - {transaction-timeout}`. By waiting for the transaction timeout window to close, the client can be sure the data being returned is complete and in order, and new data will not appear. The transaction timeout default value is 120s but this value is configurable. Implementers should check with server administrators on the correct value to use.
+| logical-id | version |  time | change-id |  change-type |
+| ---------- | ------- | ----- | --------- | ------------ |
+| patient-1  |       1 | 12:00 |         1 | CREATE |
+| patient-2  |       1 | 12:05 |         2 | CREATE |
+| patient-3  |       1 | 12:05 |         3 | CREATE |
+| patient-4  |       1 | 12:04 |         4 | CREATE |
+| patient-1  |       2 | 12:06 |         5 | UPDATE |
+| patient-3  |       2 | 12:06 |         6 | DELETE |
+
+Note how the change time for `patient-3` and `patient-4` is the same, although they have different change ids. Significantly, `patient-4` has a change time before `patient-2` even though its id is greater. This can happen if the clocks in a cluster are not perfectly synchronized. This only applies to different resources - changes can _never_ appear out of order for a specific resource because the IBM FHIR Server uses database locking to ensure consistency. 
+
+Clients must also exercise caution when reading recently ingested resources. When processing large bundles in parallel, an id may be assigned by the database but ACID isolation means that the record will not be visible to a reader until the transaction is committed. This could be up to 120s or longer if a larger transaction-timeout property has been defined. If a smaller bundle starts after the larger bundle and its transaction is committed first, its change ids and timestamps will be visible to readers before the resources from the other bundle, which will have earlier change ids and timestamps. If clients do not take this into account, they may miss some resources. This behavior is a common concern in databases and not specific to the IBM FHIR Server.
+
+To guarantee no data is skipped, clients should not process resources with a `lastModified` timestamp which is after `{current_time} - {transaction_timeout} - {max_cluster_clock_drift}`. By waiting for this time window to close, the client can be sure the data being returned is complete and in order, and new data will not appear. The default value for transaction timeout is 120s but this is configurable. A value of 2 seconds is a reasonable default to consider for `max_cluster_clock_drift` in lieu of specific information. Implementers should check with server administrators on the appropriate values to use.
```

Comment:
> clients should not process resources with a `lastModified` timestamp which is after `{current_time} - {transaction_timeout} - {max_cluster_clock_drift}`

can we be more specific with this recommendation.  how would they accomplish this?  just stop resolving the 'next' link once they get to a resource on this window, then re-issue the same query until the window is passed?

Reply:
that seems a bit complicated...maybe we could introduce a query parameter which will cause the server to only return resources with a lastModified instant that is less than `{current_time} - {transaction_timeout} - {max_cluster_clock_drift}`?

Reply:
Raised #2039 for this.